mypy/typeanal.py

```diff
@@ -731,6 +733,23 @@ def visit_callable_type(self, t: CallableType) -> TypeVarList:
             return []
 
 
+def has_any_from_silent_import(t: Type) -> bool:
```

Comment:
Responded to this in this discussion: https://github.com/python/mypy/pull/3405#discussion_r118377736